packages/backend/src/core/discovery/handlers/user/index.ts

```diff
@@ -64,6 +69,13 @@ export function getUserHandler(
       return new AccessControlHandler(field, definition, abi, logger)
     case 'arrayFromOneEvent':
       return new ArrayFromOneEventHandler(field, definition, abi, logger)
+    case 'arrayFromOneEventWithArgs':
```

Comment:
add usage to Notion docs https://www.notion.so/l2beat/Handlers-8fd74177dde44b0bbc42f24f11aab812

Reply:
done: https://www.notion.so/l2beat/Array-from-one-event-with-args-5d6fc2eb50414dce867ce2009b95d89e